Dockerfile

```diff
@@ -68,6 +68,16 @@ RUN pip install wheel \
   && mv .local/bin/bandit .local/bin/bandit2 \
   && pip3 install --user bandit==${BANDIT_VERSION}
 
+# Install solidity compiler version manager
+ENV SOLC_SELECT_VERSION 0.2.1
+
+RUN pip3 install --user solc-select==${SOLC_SELECT_VERSION}
+ENV PATH="/root/.local/bin:${PATH}"
+
+# Install slither, solidity static code scanner
+ENV SLITHER_VERSION 0.8.3
+RUN pip3 install --user --no-cache-dir slither-analyzer==${SLITHER_VERSION}
```

Comment:
Can you combine these pip3 install commands (that will result in a smaller image)?